packages/ag-charts-community/src/scene/group.ts

```diff
@@ -301,10 +305,26 @@ export class Group extends Node {
         }
         if (stats) stats.nodesSkipped += skipped;
 
+        if (clipShape && clipMode === 'outside') {
```

Comment:
Do we need to add support for this behaviour to `Group`? I'd prefer not to add more concerns into the mix for `Group` if possible (also, we should implement support for layer-handing for non-BBox clips if we add it here); could we either implement this behaviour as a mixin for specific cases, or as part of `Shape` or another relevant `Node` subclass?